### DISCUSSION TITLE
What does "Files with missing rpath" mean and how can I fix this?

### DISCUSSION THREAD

**Discussion by eleanordoesntcode** [Homebrew/discussions#5773] [Writing Formulae/Casks]
### Output of `brew config`

```shell
Not applicable. Error happens on a GitHub Runner
```

### Output of `brew doctor`

```shell
Same.
```

### Description of issue

I'm currently updating the projectm formulae, which installs among other things some dynamic libraries, and I'm getting the error "Files with missing rpath:
    /opt/homebrew/Cellar/projectm/4.1.2/lib/libprojectM-4-playlist.4.1.2.dylib" at the `brew linkage --cached --test --strict projectm` step. Why does it happen and how can I fix this?

**Comment by p-linnane** [comment]
An rpath is like a map to find shared libraries. I looked at your PR, and you probably just need to add `"-DCMAKE_INSTALL_RPATH=#{rpath}"` right before `*std_cmake_args`.

**Reply by eleanordoesntcode** [reply]
Thanks a lot! I'll try this.

**Reply by eleanordoesntcode** [reply]
Sadly this hasn't solved the problem, although that's not the only problem I'm encountering with this build, so I think it'll take me a few more days to do this.
